src/components/IPScan/Actions/Group/DownloadAll/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { createSelector } from 'reselect';
+import { format } from 'url';
+
+import loadData from 'higherOrder/loadData/ts';
+import { Params } from 'higherOrder/loadData/extract-params';
+
+import Tooltip from 'components/SimpleCommonComponents/Tooltip';
+import Link from 'components/generic/Link';
+
+import { getAllResults, Jobs } from '..';
+
+import cssBinder from 'styles/cssBinder';
+
+import fonts from 'EBI-Icon-fonts/fonts.css';
+import local from '../../style.css';
+
+const css = cssBinder(fonts, local);
+
+const downloadFile = (jsonContent: Record<string, unknown>, name: string) => {
+  const downloadContent = JSON.stringify(jsonContent);
+  const blob = new Blob([downloadContent], { type: 'application/json' });
+
+  const link = document.createElement('a');
+  link.style.display = 'none';
+  link.href = URL.createObjectURL(blob);
+  link.download = name;
+
+  document.body?.appendChild(link);
+  link.click();
+
+  setTimeout(() => {
+    URL.revokeObjectURL(link.href);
+    link.parentNode?.removeChild(link);
+  }, 0);
+};
+
+type Props = {
+  jobs: Jobs;
+  remoteID?: string;
+  group: string;
+  dataURL?: string;
+};
+interface LoadedProps extends Props, LoadDataProps<string> {}
+
+const DownloadAll = ({ jobs, group, remoteID, data, dataURL }: LoadedProps) => {
+  const handleDownload = async () => {
+    downloadFile(await getAllResults(jobs, group), `${group}.json`);
+  };
+  const thereDataInServers = remoteID && data?.payload === 'FINISHED';
+  return thereDataInServers ? (
+    ['tsv', 'json', 'xml', 'gff', 'sequence'].map((type) => (
+      <li key={type}>
+        <Tooltip
+          title={
+            <div>
+              This will download the data that was originally loaded to our
+              servers. This is only available for 7 days after runnin the job.
+            </div>
+          }
+        >
```

Comment:
Minor typo, but I don't think we need a tooltip here, tbh.

```suggestion
        <Tooltip
          title={
            <div>
              This will download the data that was originally loaded to our
              servers. This is only available for 7 days after running the job.
            </div>
          }
        >
```